src/components/CancelSwitchPopup/CancelSwitchPopup.js

```diff
@@ -170,8 +175,8 @@ const CancelSwitchPopup = ({
             <Button
               theme="confirm"
               onClickFn={() => {
-                updateList();
-                hideInnerPopup();
+                dispatch(hidePopup());
+                dispatch(updateList());
```

Comment:
could we make one action that do both?

Reply:
hidePopup is action from `popupSlice` and updateList is from the `planDetailsSlice` so dont know if we can/should combine them in one action. What do you think?